app/models/enumerators/nasa_viirs_fire_alerts/supported_attributes.py

```diff
@@ -32,17 +32,10 @@ class SupportedAttribute(Enum, init="value aggregation_rule"):  # type: ignore
         rule('round(avg("bright_ti5__k"),3)', "bright_ti5__K"),
     )
     FRP_MW = ("frp__MW", rule('sum("frp__MW")', "frp__MW"))
-    UMD_TREE_COVER_DENSITY_2000__THRESHOLD = (
-        "umd_tree_cover_density_2000__threshold",
-        rule(
-            'max("umd_tree_cover_density_2000__threshold")',
-            "umd_tree_cover_density_2000__threshold",
-        ),
-    )
```

Comment:
I forget why this is unnecessary, is it because it's not actually used anywhere?

Reply:
Good question, I discussed this earlier with @jterry64 and the `umd_tree_cover_density_2000__threshold` is the same as `umd_tree_cover_density__threshold`. Since the _2000_ version of the attribute was just recently deployed to production by me, I think it's safe to say that there are no consumers of this attribute and it can go away.